ec2/sc-ec2-windows-jumpcloud-v1.0.0.yaml

```diff
@@ -235,6 +250,62 @@ Resources:
                   - ' -Project $env:PROJECT'
                   - ' -OwnerEmail $env:OWNER_EMAIL'
                   - ' > C:\scripts\tag-instance-volume.log'
+        tag_instance:
+          files:
+            'c:\\scripts\\tag_instance.ps1':
```

Comment:
consider putting this script into Sage-Bionetworks/infra-utils so it can be re-used in ec2/sc-ec2-windows-jumpcloud-development.yaml ?

Reply:
Sure, but how about we do a separate PR to move both scripts I've written?

Reply:
sounds good